### PR TITLE
add rename column executor

### PR DESCRIPTION
**Pull request by sxzh93**

### CODE REVIEW

File: src/include/executor/alter_executor.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ protected:
+  bool DInit();
+
+  bool DExecute();
+
+  bool RenameColumn(const planner::RenamePlan &node,
+                    concurrency::TransactionContext *txn);
+
+ private:
+  ExecutorContext *context_;
```
[the remark this answers is not in the document]
Comment (Dingshilun) [reply src/include/executor/alter_executor.h]: I think context might be needed in the future as new alter commands might be added

[the remark this answers is not in the document]
Comment (DeanChensj) [reply src/include/executor/alter_executor.h]: We can get executor_context from the base class directly, no need to store another copy.

File: src/common/internal_types.cpp [hunk 2]
```diff
@@ -417,6 +417,39 @@ std::ostream &operator<<(std::ostream &os, const DropType &type) {
...
 }
 
+std::string AlterTypeToString(AlterType type) {
+  switch (type) {
+    case AlterType::INVALID: {
+      return "INVALID";
+    }
+    case AlterType::RENAME: {
+      return "RENAME";
+    }
```
[the remark this answers is not in the document]
Comment (sxzh93) [reply src/common/internal_types.cpp]: see latest commit

File: src/executor/alter_executor.cpp [hunk 3]
```diff
@@ -0,0 +1,70 @@
+#include "executor/alter_executor.h"
```
[the remark this answers is not in the document]
Comment (sxzh93) [reply src/executor/alter_executor.cpp]: resolved

File: src/include/executor/executors.h [hunk 4]
```diff
@@ -38,3 +38,4 @@
 #include "executor/projection_executor.h"
 #include "executor/seq_scan_executor.h"
 #include "executor/update_executor.h"
+#include "executor/alter_executor.h"
```
Comment (DeanChensj) [inline src/include/executor/executors.h]: Include file in alphabet order.. Lol

**Pull request merged**